### PR TITLE
Trail review

### PR DESCRIPTION
**Pull request by MariamOsman11**
# What I did
-Create a new empty repo.
-Copy your code from the project that you completed during the Admissions Trial
-Create a new branch for your copied code.
-Open a pull request
-Play the role of the Code Reviewer:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+                With the keyword var. For example,</p>
+                <code>var x = 42.</code>
+            <p>This syntax can be used to declare both local and global variables.By simply assigning it a value. For example,</p>
+                <code>x = 42.</code>
+            <p>This always declares a global variable. It generates a strict JavaScript warning. You shouldn't use this variant.</p>
+
+                      <p> With the keyword let. For example,</p>
+                      <code>let y = 13.</code>
+
+                      <p>This syntax can be used to declare a block scope local variable. See Variable scope below.</p>
```
Comment (MariamOsman11) [inline index.html:109]: I suggest that you remove the blank spaces, to make your code cleaner.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+              <li>Boolean. true and false.</li>
+              <li>null. A special keyword denoting a null value. Because JavaScript is case-sensitive, null is not the same as Null, NULL, or any other variant.</li>
+              <li>undefined. A top-level property whose value is undefined.</li>
+              <li>Number. 42 or 3.14159.</li>
+              <li>String. "Howdy"</li>
+              <li>Symbol (new in ECMAScript 2015). A data type whose instances are unique and immutable</li>
+            </ul>
+          </li>
+          <li>and Object</li>
+        </ul>
```
Comment (MariamOsman11) [inline index.html:167]: I suggest you remove the indentation from these lines  for easier readability.

**Review by MariamOsman11 — COMMENTED** [review]
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
image


Highlights
Well-structured file and folder structure ✅
Used correctly Github Flow ✅
Design looks similar to Freecode acadamy ✅
Well implemented the features ✅
Required Changes ♻️
Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @skyv26 in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by divinecharlotte — APPROVED** [review]
# Hello @MariamOsman11    😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- No linters errors ✔️
- good communication while giving feedback👍 





## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**